test/unittests/validation_test.cpp

```diff
@@ -406,7 +406,7 @@ TEST(validation, load_alignment)
         {Instr::f64_load, 3},
     };
 
-    for (const auto test_case : test_cases)
+    for (const auto& test_case : test_cases)
```

Comment:
Found by XCode 12, but I removed usage of this compiler because of issues with container overflow detection.